ffi/SDL2_0.lua

```diff
@@ -190,6 +190,7 @@ function S.setWindowFullscreen(full_screen)
 end
 
 function S.destroyTexture(texture)
+    ffi.gc(texture, nil)
```

Comment:
Why is that one necessary? Is there an implicit finalizer on this somehow?

Reply:
Nope, good catch: leftover from before I added a finalizer for the renderer (which takes care of the texture too).